src/NUnitFramework/tests/Constraints/DelayedConstraintTests.cs

```diff
@@ -40,7 +40,7 @@ public class DelayedConstraintTests : ConstraintTestBase
         // constructing the constraint class, and verify that they work.
 
         private const int DELAY = 100;
-        private const int AFTER = 300;
+        private const int AFTER = 500;
```

Comment:
Azure DevOps' macOS has been failing `ThatBlockingDelegateWhichSucceedsWithPolling_ReturnsEarly` on almost every build for a while. 300 ms is just too close to 100 ms for extremely shared servers.